eZ/Publish/Core/MVC/Symfony/Templating/Twig/Extension/ContentExtension.php

```diff
@@ -78,6 +78,10 @@ public function getFunctions()
                 array($this, 'getTranslatedFieldValue')
             ),
             new Twig_SimpleFunction(
+                'ez_field_content',
```

Comment:
Come to think of it, yeah, ez_field sounds more reasonable as it returns exactly that.